### PR TITLE
 [MIG] partner_payment_return_risk > account_payment_return_financial_risk: Migration to 11.0

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# list the OCA project dependencies, one per line
+# add a github url if you need a forked version
+account-payment
+https://github.com/tecnativa/credit-control/tree/11.0-mig-partner_financial_risk
```
Comment (pedrobaeza) [inline oca_dependencies.txt]: To be removed before merging

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
You have to rename all the occurrences of `partner_payment_return_risk` to `account_payment_return_financial_risk` on translation files.

**Comment by pedrobaeza** [comment]
@carlosdauden tests are not adjusted for v11. Please fix that

File: account_payment_return_financial_risk/tests/test_payment_return_risk.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                'name': 'Test line',
+                'price_unit': 50,
+                'quantity': 10,
+            })]
+        })
+        cls.reason = cls.env['payment.return.reason'].create({
+            'code': 'RTEST',
+            'name': 'Reason Test'
+        })
+        cls.invoice.signal_workflow('invoice_open')
```
Comment (cubells) [inline account_payment_return_financial_risk/tests/test_payment_return_risk.py]: You must change `cls.invoice.signal_workflow('invoice_open')` by `cls.invoice.action_invoice_open()`

**Review by cubells — CHANGES_REQUESTED** [review]

**Comment by carlosdauden** [comment]
@cubells All green now

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Merging as @cubells's comments have been honored.

**Comment by cubells** [comment]
thanks @carlosdauden

**Pull request merged**